okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -85,7 +85,13 @@
     private static final Dispatcher dispatcher;
 
     /** Shared connection pool. */
-    private static final ConnectionPool connectionPool = new ConnectionPool(100, 10, TimeUnit.MINUTES);
+    private static final ConnectionPool connectionPool = new ConnectionPool(
+            1000,
```

Comment:
In WC's ServerConfiguration, I think we have an upper bound of 2560 concurrent connections.  If client nodes start maxing out their connection pools and try to open 1000 each, doesn't this mean that 2 client nodes could hog an entire server node?

Is this a risk we're willing to take (i.e. because we think these connections will not be maxed out the whole time, and hope that servers are horizontally scaled sufficiently to accommodate their possible clients?)

```java
    /** The maximum number of threads to use for requests. */
    @Value.Default
    @JsonProperty("max-threads")
    default int maxThreads() {
        return Math.max(256, 32 * processors());
    }

    @Value.Default
    @JsonProperty("max-connections") 
    default int maxConnections() {
        return 10 * maxThreads();
    }
```

Reply:
I don't think this limit stops us from making more requests, it only stops us from saving the idle connections. In the heavily loaded case we should try to reuse connections as much as possible to avoid handshake work.

Reply:
argh you're absolutely right - that first param is `maxIdleConnections`! OK totally fine with dialling this up, as this isn't going to suddenly unleash any floodgates that were being capped. 

So we're really only expecting this to eliminate some possible blockers for very bursty traffic.